c++/src/io/OutputStream.cc

```diff
@@ -96,6 +96,9 @@ namespace orc {
     dataBuffer->resize(0);
     return dataSize;
   }
+  void BufferedOutputStream::suppress() {
```

Comment:
Could you add an empty line before this?

Reply:
done.